docs/DeveloperGuide.adoc

```diff
@@ -279,51 +279,203 @@ Refer to the guide <<DevOps#, here>>.
+
+|`* *` |diabetic patient with a family |have a user-friendly app with natural commands that helps me manage my medical data and appointments on my own |free the burden I have on my family
+
+|`*` |diabetic patient in a community of diabetic patients |have a standardised means of comparing our activities via a social network | learn from my peers, encourage and be encouraged through this difficult journey.
 
-_{More to be added}_
+|`*` |careless user  |undo my most recent actions |easily make necessary amendments and input the correct commands
+
+|`*` |a diabetic patient who has many medical receipts - and is not very good at mathematics |have a simple calculator that is always easily accessible |instantly calculate all my medical costs when needed
+
+|`*` |an obese working adult at high risk of diabetes |start monitoring my diet |minimise my risk of having diabetes
+
+|`*` |medical consultant |export my patient's health data  |save my time
+|=======================================================================
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `Sugar Mummy` and the *Actor* is the `user`, unless specified otherwise)
+
+=== Use case: Add blood sugar record
+*MSS*
+
+1.  User requests to add a blood sugar record
+2.  System adds the blood sugar record
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The record is incomplete or passed invalid arguments.
++
+[none]
+** 1a1. System shows an error message.
++
+Use case resumes at step 1.
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Schedule a medical appointment
+*MSS*
+
+1.  User requests to add a medical appointment
+2.  System adds the medical appointment
+3.  System notifies user of upcoming medical appointment beforehand
+4.  User acknowledges the notification and attends medical appointment on schedule
++
+Use case ends.
+
+*Extensions*
 
+[none]
+* 1a. The appointment is incomplete or passed invalid arguments.
++
+[none]
+** 1a1. System shows an error message.
++
+Use case resumes at step 1.
++
+[none]
+* 3a. User snoozes the notification.
++
+[none]
+** 3a1. System waits for snooze time to elapse.
++
+Use case resumes at step 3.
+
+[discrete]
+=== Use case: Delete blood sugar record
+*MSS*
+
+1.  User requests list of blood sugar records
+2.  System shows a list of blood sugar records
+3.  User requests to delete a specific blood sugar record in the list
+4.  System deletes the blood sugar record
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The list is empty.
++
+Use case ends.
+
+* 3a. The given index is invalid.
++
+[none]
+** 3a1. System shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Recommend diabetes-friendly food
+*MSS*
+
+1.  User requests for a diabetes-friendly food item
+2.  System shows a diabetes-friendly food item
+3.  User likes the recommendation
```

Comment:
3 and 3a can be moved to 2a and 2b as the current implementation implies that the user has to like the recommendation before he can dislike it.